app/serializers/insecure_user_serializer.rb

```diff
@@ -0,0 +1,7 @@
+class InsecureUserSerializer < ActiveModel::Serializer
```

Comment:
Ok, there was no migration created for the serializers, so I'm good to delete it. 

I updated the code per your feedback and merged with master. Sorry I forgot the pull request. This workflow is still taking some getting used to. 



> On Dec 29, 2015, at 9:15 PM, David Roberts <notifications@github.com> wrote:
> 
> In app/serializers/insecure_user_serializer.rb:
> 
> > @@ -0,0 +1,7 @@
> > +class InsecureUserSerializer < ActiveModel::Serializer
> That is a good question. Did you create a migration that specified InsecureUser? If not, you are good to go!
> 
> For the record, these serializers are purposely abstracted from what's in the database. The model represents what is in the database, and the serializer is an alternative representation of the model
> 
> —
> Reply to this email directly or view it on GitHub.
>